doc/rfcs/0002-support-run-component-without-tiup.md

```diff
@@ -0,0 +1,23 @@
+# Support Run Component without TiUP
+
+## Summary
+
+Allow users to only use TiUP during installation and upgrade, without needing to invoke TiUP while using the component.
+
+## Motivation
+
+- Some users do not like the mandatory update checks and additional outputs by TiUP before invoking the components.
+- For programs that are not only distributed through TiUP, the existing usage patterns have led to documentation fragmentation.
+
+## Detailed design
+
+1. Add `tiup link <component>[:version]` command to add soft link to $TIUP_HOME/bin
+
+2. Add `tiup unlink <component>[:version]` command to delete soft link from $HOME/bin
+
+3. Add `--link` flag to `tiup install` and `tiup update` command to link while install/update
+
+4. Mark these command and flag as experimental feature and we keep the old behavior as the default usage method.
+Warn: Users may need to manually set environment variables for certain components, such as ctl.
```

Comment:
Would this conflict with `etcdctl` from the system installation?

```
$ ls -1 .tiup/components/ctl/v7.2.0/
binlogctl
cdc
ctl
etcdctl
pd-ctl
tidb-ctl
tidb-lightning-ctl
tikv-ctl
```

Would the user do a `tiup link ctl:v7.2.0` and then get links for all of the above binaries?

Reply:
Would this conflict with etcdctl from the system installation?